index.html

```diff
@@ -1,19 +1 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>name</title>
-</head>
-<body>
-    <main>
-        <div>
-            <p>This is a training Demo for the goons</p>
-
-            <p>Name: Adetoyinbo Kolade</p>
-            <p>Name: Adeleye ifedayo</p>
-            <p>Name: Amarachi Onyendi</p>
-        </div>
-    </main>
-</body>
-</html>
\ No newline at end of file
+ pus
```

Comment:
@Theo-flux 
Why do we have this changes here?

Reply:
Oh, It must have been a mistake. Reverting changes now.